### PR TITLE
Create jwt-validation.mdx

### PR DESCRIPTION
**Pull request by mkarnowski**
initial jwt validation document

### CODE REVIEW

File: docs/http/jwt-validation.mdx [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+You may specify multiple audience claims to be used for JWT validation. JWT validation will require one of the audience claims to be present within the JWT.
+
+### Multiple JWS
+
+You have the ability to provide multiple JWKS urls. During JWT validation the list of JWKS provided will be used to validate the JWT.
+
+### Multiple Tokens
+
+If multiple tokens are defined within the HTTP configuration parameter, all tokens must be present in the request. I all tokens are not present a 401 status code will be returned.
```
Comment (wdawson) [inline docs/http/jwt-validation.mdx:66]:
```suggestion
If multiple tokens are defined within the HTTP configuration parameter, all tokens must be present in the request. If all tokens are not present a 401 status code will be returned.
```

File: docs/http/jwt-validation.mdx [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+### Upstream Headers {#upstream-headers}
+
+This module does not add any upstream headers.
+
+### Errors
+
+| Code                | HTTP Status | Error                                                                |
+| ------------------- | ----------- | -------------------------------------------------------------------- |
+| no ngrok error code | `401`       | This error is returned if the request is unauthorized.               |
+| no ngrok error code | `403`       | This error is returned if the JWT cannot be validated by the issuer. |
```
Comment (wdawson) [inline docs/http/jwt-validation.mdx:88]: I believe we have several error codes for various different problems: https://github.com/ngrok-private/ngrok/blob/main/config/src/errors/mw.ts#L1019

File: docs/http/jwt-validation.mdx [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+is considered validated if it presents a JWT signed by any of the
+specified issuers.
+
+### Multiple Audience Claims
+
+You may specify multiple audience claims to be used for JWT validation. JWT validation will require one of the audience claims to be present within the JWT.
+
+### Multiple JWS
+
+You have the ability to provide multiple JWKS urls. During JWT validation the list of JWKS provided will be used to validate the JWT.
```
Comment (wdawson) [inline docs/http/jwt-validation.mdx:62]: do we want to talk about the allowed algorithms as well?

**Review by wdawson — APPROVED** [review]
Left some comments but I'm happy with this as a starting place.

File: docs/http/jwt-validation.mdx (hunk 3 above)
Comment (mkarnowski) [reply docs/http/jwt-validation.mdx]: added mention

File: docs/http/jwt-validation.mdx (hunk 2 above)
Comment (mkarnowski) [reply docs/http/jwt-validation.mdx]: @wdawson Do those errors get exposes to end users?  In my testing I've only been able to raise 3401 and 3403.

**Review by russorat — APPROVED** [review]
lgtm!

File: docs/http/jwt-validation.mdx (hunk 2 above)
Comment (wdawson) [reply docs/http/jwt-validation.mdx]: ah! looks like no. we can keep that for security or change it to balance more towards devex I think

**Pull request merged**